Dockerfile

```diff
@@ -3,12 +3,15 @@ WORKDIR /usr/src/app/
 
 COPY .babelrc package.json package-lock.json ./
 RUN apk add --no-cache --virtual .gyp python3 make g++ &&\
-    npm install &&\
+    apk add dumb-init &&\
```

Comment:
dumb-init was added because nodejs is not meant to run as PID 1.
For instance it does not respond to sigint (ctrl-c) signals to stop the container.
dumb-init is a simple process supervisor and init system designed to run as PID 1 inside minimal container environments e.g. docker, which is perfect for our project